crossopt/assignment3/unittests.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+import unittest
+from assignment3.dictionary import Dictionary
+from assignment3.wordsearch import Grid
+from assignment3.wordsearch import findMatch, findMatches
+
+
+class TestDictionary(unittest.TestCase):
+    def test_isprefix(self):
+        dictionary = Dictionary(['CAR', 'CART', 'CARD', 'CAT'])
+        self.assertEqual(dictionary.isPrefix('A'), 0)
+        self.assertEqual(dictionary.isPrefix('car'), 0)
+        self.assertEqual(dictionary.isPrefix('CARDS'), 0)
+        self.assertEqual(dictionary.isPrefix(''), 1)
+        self.assertEqual(dictionary.isPrefix('CARD'), 1)
+        self.assertEqual(dictionary.isPrefix('CAR'), 1)
+
+    def test_isword(self):
+        dictionary = Dictionary(['CAR', 'CART', 'CARD', 'CAT'])
+        self.assertEqual(dictionary.isWord('A'), 0)
+        self.assertEqual(dictionary.isWord('C'), 0)
+        self.assertEqual(dictionary.isWord('car'), 0)
+        self.assertEqual(dictionary.isWord('CARDS'), 0)
+        self.assertEqual(dictionary.isWord(''), 0)
+        self.assertEqual(dictionary.isWord('CARD'), 1)
+        self.assertEqual(dictionary.isWord('CAR'), 1)
+
+
+class TestGrid(unittest.TestCase):
+    def test_getitem(self):
+        array = [['A', 'A', 'R'], ['T', 'C', 'D']]
```

Comment:
WIthout changing the implementation of the current Grid, I think you can replace:
 [['A', 'A', 'R'], ['T', 'C', 'D']]

with 

['AAR', 'TCD']

List a list, a string is iterable.